### PR TITLE
Windows Build Doc & Package.json for Windows

### PR DESCRIPTION
**Pull request by mayurdw**
The current JSON file cause errors because of the file separators. I added new commands inside the file specially for Windows. Every windows command is with a prefix 'windows-'
The build installation.md file also reflects that and follow the steps I took to build
PR for Issue #18

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,18 +42,26 @@
     "underscore": "1.8.x"
   },
   "scripts": {
+    "windows-start":"npm run client-windows",
```
Comment (dweremeichik) [inline package.json]: Is client-windows correct here? Shouldn't it be windows-client?

File: docs/guide/contributing/installation.md [hunk 2]
```diff
@@ -35,3 +35,33 @@ If you are looking to simply install Nylas-Mail on your system (and are not look
...
+5.  Install the necessary node packages using:
+    ```bash
+    npm install
+    ```
+6.  Assuming node was able to install the appropriate dependencies without error, you can build the client app with:
+    ```bash
+    npm run windows-build-client
+    ```
+7. If there are no errors in the previous two steps, start the app with:
+  ```bash
```
Comment (dweremeichik) [inline docs/guide/contributing/installation.md]: Indentation.

**Comment by mayurdw** [comment]
@dweremeichik I made the fixes and pushed them. Regarding your question 
>Quick question, if you use git bash or cygwin, do you need this fix?
I don't know honestly. I used Command Prompt for my build. I don't know what @seesemichaelj used.

**Comment by seesemichaelj** [comment]
@dweremeichik I'm going to give this all a try and will get back tomorrow with my review

**Comment by seesemichaelj** [comment]
Dependencies are not stylized in a list: 
image

File: docs/guide/contributing/installation.md [hunk 3]
```diff
@@ -35,3 +35,33 @@ If you are looking to simply install Nylas-Mail on your system (and are not look
     npm start
     ```
     * If an error was thrown during the build process, please make sure all of your dependencies were installed in step 2.
+
+## Windows:
+1. Install Node.js version 6.9.x ( Can use Windows [NVM](https://github.com/coreybutler/nvm-windows))
```
Comment (seesemichaelj) [inline docs/guide/contributing/installation.md]: remove space before "Can"

**Comment by seesemichaelj** [comment]
Replicating your instructions cannot execute `npm install` properly. the Isomorphic library doesn't build

**Comment by mayurdw** [comment]
What is the error you get? @seesemichaelj

**Comment by seesemichaelj** [comment]
It could not find the Windows SDK 8.1. I guess I didn't check that when installing the community version.

**Comment by mayurdw** [comment]
You mean the VS community installation?

**Comment by seesemichaelj** [comment]
Yes; I believe the Windows 8.1 SDK was not installed by default when installing VS Community. It was under "Windows 8.1 and ... Tools" or something like that

**Comment by seesemichaelj** [comment]
Yup that definitely was the error; all of the isomorphic core stuff is building now

**Comment by seesemichaelj** [comment]
we could get away from multiple build commands by using node scripts like https://shapeshed.com/writing-cross-platform-node/#scripts-in-package-json

**Comment by seesemichaelj** [comment]
Now I'm getting the error `C:\nylas-mail\packages\client-app\node_modules\npm\node_modules\node-gyp\src\win_delay_load_hook.c(34): error C2373: '_ _pfnDliNotifyHook2': redefinition; different type modifiers [C:\nylas-mail\packages\client-app\node_modules\sqlite3\bui ld\deps\sqlite3.vcxproj]`

**Comment by seesemichaelj** [comment]
This PR is no longer applicable due to the conflicts of the new merge. Please rebase to the new `master` branch and bring over your fixes. You can cherry-pick the commits. Please see the `v2.1.0` branch to see the old master branch. Thanks!

There are also some discrepancies previously mentioned that need to be addressed.

**Pull request closed without merging**